oaknational/Oak-Web-Application
# chore: update to subject listing mv 2

## Description

- Connects mv subject listing 2 
- fixes missing unit variant programme factors on subject listing page

## Issue(s)

Fixes #LESQ-88

## How to test

1. Go to subject listing (2023) click on history - should take you to programme listing page not unit listing
